config/locales/controllers/admin/en.yml

```diff
@@ -8,7 +8,7 @@ en:
     badge_achievements_controller:
       deleted: Badge achievement has been deleted!
       rewarded: Badges are being rewarded. The task will finish shortly.
-      congrats: Congrats!
+      congrats: Congrats on your achievement! 🎉 And thank you for being such a vital part of the %{community} Community!
```

Comment:
`Settings::Community.community_name` includes the word `Community` (i.e. DEV Community), so this should be removed from the copy.

Reply:
Good catch - thanks!